docs/_tooling/extensions/score_draw_uml_funcs/__init__.py

```diff
@@ -21,21 +21,21 @@
 
 and all applicable linkages between them.
 
-It provieds this functionality by adding classes to `needs_render_context`, which then can be invoked inside
-'needarch' and 'needuml' blocks in rst files.
+It provides this functionality by adding classes to `needs_render_context`,
+which then can be invoked inside 'needarch' and 'needuml' blocks in rst files.
 """
 
 import hashlib
 import time
-
 from functools import cache
 from pathlib import Path
-from sphinx.application import Sphinx
-from sphinx_needs.logging import get_logger
+
+from sphinx.application import Sphinx  # type: ignore
+from sphinx_needs.logging import get_logger  # type: ignore
```

Comment:
ignore should not be necessary with `docs:ide_support`. Works for me. That was the entire point of `ide_support`.

Reply:
Let's merge and have a call later. It should work, and it is important for type checking.